time stamp

string subtask_success_status
string task_success_status


## Debug values | also for statistics

uint32 current_object
uint32 current_tray
uint32 success_tray
uint32 task_counter
uint32 subtask_counter

## mainly used for statistics

uint32  failed_subtasks
uint32  successful_subtasks
float64 subtask_time_seconds
float64 task_combined_subtask_time_seconds
float64 percentage_successful_subtasks
string  who_succeeded
uint32  task_warnings_received
uint32  successful_tasks_cnt
uint32  failed_tasks_cnt
float64 percentage_successful_tasks
